app/navigation/Navigator.scala

```diff
@@ -460,11 +461,17 @@ class Navigator @Inject() () {
       case Some(false) => ImportingGoodsController.onPageLoad(userAnswers.draftId)
     }
 
-  private def requiredInformationPage(userAnswers: UserAnswers): Call      =
+  private def requiredInformationPage(userAnswers: UserAnswers): Call =
     userAnswers.get(RequiredInformationPage) match {
       case None    => RequiredInformationController.onPageLoad(userAnswers.draftId)
       case Some(_) => ImportGoodsController.onPageLoad(NormalMode, userAnswers.draftId)
     }
+
+  private def traderAgentRequiredInformationPage(userAnswers: UserAnswers): Call =
+    userAnswers.get(TraderAgentRequiredInformationPage) match {
+      case Some(_) => ImportGoodsController.onPageLoad(NormalMode, userAnswers.draftId)
+    }
```

Comment:
The `None` case needs to be handled here too. I presume in this instance it would look like:

```scala
case None => TraderAgentRequiredInformationController.onPageLoad(userAnswers.draftId)
```